src/content/docs/index.mdx

```diff
@@ -0,0 +1,21 @@
+---
+title: Starlight 🌟 Build documentation sites with Astro
+description: Starlight helps you build beautiful, high-performance documentation websites with Astro.
+template: splash
+editUrl: false
```

Comment:
Is there a way to enable this? Would be cool to show an edit button.

Reply:
I setting it to `true` works.